app/javascript/components/student/editor/TestRunSummary.tsx

```diff
@@ -1,47 +1,34 @@
 import React, { useState, useEffect, useCallback, useRef } from 'react'
-import { Submission, TestRunStatus } from '../Editor'
+import { Submission, TestRun, TestRunStatus } from '../Editor'
 import { TestRunChannel } from '../../../channels/testRunChannel'
 import { TestRunSummaryContent } from './TestRunSummaryContent'
 import { fetchJSON } from '../../../utils/fetch-json'
 
-export type TestRun = {
```

Comment:
This component doesn't maintain any state anymore. We just update the parent component's state from here now.